Node/src/botbuilder-teams.d.ts

```diff
@@ -34,444 +34,288 @@
 
 import * as builder from 'botbuilder';
 
-/**
- * @class
- * Initializes a new instance of the ChannelInfo class.
- * @constructor
- * A channel info object which describes the channel.
- * @member {string} [name] Name of the channel
- *
- * @member {string} [id] Unique identifier representing a channel
- *
- *
- */
-export interface ChannelInfo {
-  name?: string;
-  id?: string;
-}
-
-/**
- * @class
- * Initializes a new instance of the ChannelAccount class.
- * @constructor
- * A channel account object which describes the member.
- * @member {string} [id] Unique identifier representing a member
- *
- * @member {string} [objectId] User Id
- *
- * @member {string} [name] The display name for the member
- *
- * @member {string} [givenName] Name of the member
- *
- * @member {string} [surname] Name of the member
- *
- * @member {string} [userPrincipalName] Name of the member
- *
- * @member {string} [email] Email of the member
- *
- *
- */
+/** Information about a Microsoft Teams user. */
```

Comment:
Do we change comments style in the future?

Reply:
This style works better with intellisense. When you put the comment on the member, hovering over the usage gives you a nice tooltip w/ documentation. Previously you didn't get anything except when you hover over the type name.